### ISSUE TITLE
order banners

### ISSUE THREAD

**Issue by ashumz** [ashumz/citycamp#17]
![officemax-banner](https://cloud.githubusercontent.com/assets/100216/7924406/e880cbee-0888-11e5-97aa-c6b2995da361.png)

**Comment by ashumz** [comment]
the long banners (for welcome and sponsors) are $49.99 at Office Max (We would need stands for them)

**Comment by dtraleigh** [comment]
Chris and I talked about this when visiting the commons. Where do we see putting these? Last year, we hung the banner with string between two columns right outside the lecture hall that served as the main entrance. We didn't quite see an easy way to replicate that at the Commons. Inside, we definitely need something that will hold it up rather than tie it somewhere with string.

**Comment by ashumz** [comment]
Thanks for the info @dtraleigh. I'm cool with having a welcome sign instead if we can find another easel or stand for it.

OfficeMax has 24x36 signs -- that's the size that we'd need stands for. 

We had just a welcome sign and 2 sponsor signs planned-- and a banner. But if there is nowhere to hang a banner out front, we can just do a 4th sign with the citycampnc logo to put out front of the building instead of a banner. Sound like a better idea from what you saw? (I have not been  able to do a walk through unfortunately)

**Comment by dtraleigh** [comment]
For past events, I've used similar signs that can go into stakes that you
can push into the ground. Would that suffice? Does Office Max have them?

The Wake County Commons is a more of a campus-like setting so with a sign
out front of the building, no one would see it because of the lush greenery
all around. There's a roundabout in front for easy unloading/loading as
well as some handicap spaces but the general parking lot is in another
area. (but close) You just can't see everything when you enter so signs in
a few spots will help folks that have never been there.

There are also a few other buildings that as part of the campus so it could
be confusing. I want to try and make that clearer with a "Park like a pro
at CityCamp" blog post.

**Comment by jhibbets** [comment]
I was thinking that at that roundabout, we have a sign that says

CityCamp NC

Parking -->
<-- Event

**Comment by ChrisTheDBA** [comment]
Did we settle on non doing a banner because there was no place to hang it?

**Comment by ashumz** [comment]
yep - no banner. But let's use the vertical logo sign that Carsi made to create just a sign that says CityCampNC for the location @jhibbets suggests 

cc @ChrisTheDba @dtraleigh
